rfc/rfc022-discovery-service.md

```diff
@@ -116,12 +124,14 @@ with the following additional requirements:
 
 - it MUST specify `RetractedVerifiablePresentation` as type, in addition to the `VerifiablePresentation`.
 - it MUST contain a `retract_jti` JWT claim, containing the `jti` of the presentation to retract.
+- it MUST contain a `retract_hash` JWT claim, containing the sha256 of the presentation to retract.
```

Comment:
why? VP ID is under control of the signer, and should be unique?

Reply:
obsolete